lib/node/nodes/centroid.js

```diff
@@ -1,17 +1,32 @@
 'use strict';
 
 var Node = require('../node');
+var debug = require('../../util/debug')('analysis:centroid');
 
 var TYPE = 'centroid';
 
 var PARAMS = {
     source : Node.PARAM.NODE(Node.GEOMETRY.ANY),
     category_column : Node.PARAM.NULLABLE(Node.PARAM.STRING()),
-    aggregation: Node.PARAM.NULLABLE(Node.PARAM.STRING(), 'count'),
-    aggregation_column: Node.PARAM.NULLABLE(Node.PARAM.STRING())
+    aggregation: Node.PARAM.NULLABLE(Node.PARAM.ENUM('avg', 'count', 'max', 'min', 'sum'), null),
```

Comment:
here we need a nullable enum, if `aggregation` comes with `null` value, we shouldn't set `count` as default, should we?